backend/spec/features/admin/orders/line_items_spec.rb

```diff
@@ -27,7 +27,7 @@
           expect(page).to have_content("10")
         end
         within '.line-item-total' do
-          expect(page).to have_content("$100.00")
+          expect(page).to have_content("$199.90")
```

Comment:
:+1: makes sense I ran into a similar spec issue recently.  Not sure why we don't default line item factory to be same price as variant.